src/Core/src/Core/IRefreshView.cs

```diff
@@ -21,5 +22,15 @@ public interface IRefreshView : IView
 		/// The scrollable content to refresh.
 		/// </summary>
 		IView Content { get; }
+
+		/// <summary>
+		/// Gets the command to be executed when the refresh is triggered.
+		/// </summary>
+		ICommand Command { get; }
```

Comment:
We don't need to pull `ICommand` into core for this. This can instead be something like `void UserRefreshing` and the commanding is all handled in controls.

Similar to `IButton` https://github.com/dotnet/maui/blob/3b08c20dc691c0d6314e47d4431ead56048e653a/src/Core/src/Core/IButton.cs#L6